lib/util.js

```diff
@@ -110,6 +108,21 @@ module.exports = {
       .replace(/(\{[^\/\{\}]+\})(?!\})/g, '{$1}');
   },
 
+  /**
+  * Get the format of content type header
+  * @param {string} cTypeHeader - the content type header string
+  * @returns {string} format of content type header
+  */
+  getContentTypeHeaderFormat: function(cTypeHeader) {
```

Comment:
we already have APP_JSON etc. constants defined at the top. Let's use those instead of `json` and `xml` here.

Reply:
@abhijitkane Done....please check.